crates/re_log_types/src/path/entity_path.rs

```diff
@@ -117,11 +117,27 @@ impl EntityPath {
     ///
     /// The file path separators will NOT become splits in the new path.
     /// The returned path will only have one part.
-    #[cfg(not(target_arch = "wasm32"))]
     pub fn from_file_path_as_single_string(file_path: &std::path::Path) -> Self {
         Self::from_single_string(file_path.to_string_lossy().to_string())
     }
 
+    /// Treat the file path as an entity path hierarchy.
+    ///
+    /// The file path separators will become splits in the new path.
+    pub fn from_file_path(file_path: &std::path::Path) -> Self {
+        use clean_path::Clean as _;
+        Self::new(
+            file_path
+                .clean()
+                .iter()
+                // NOTE: This can only happen when hitting the leading `/` in an absolute path, and
+                // we currently don't support entity paths with leading `/`s.
```

Comment:
So... where did we land on this in the end? If we're going for file-like paths for our entity paths, it would make sense to allow leading `/`s, right?

Reply:
We unfortunately didn't quite decide on what to do with leading slashes in the end unfortunately. let's talk about it at standup!

Reply:
The final is yes, need to reflect that in the code